ArcadiaParties-SPA/src/app/app.module.ts

```diff
@@ -1,27 +1,34 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { StoreModule } from '@ngrx/store';
-import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { BrowserModule } from "@angular/platform-browser";
```

Comment:
Quotes should be single, as you specified it in editorconfig.

Reply:
Quotes fixed.